board/safety/safety_subaru.h

```diff
@@ -35,6 +35,9 @@ AddrCheckStruct subaru_l_rx_checks[] = {
 };
 const int SUBARU_L_RX_CHECK_LEN = sizeof(subaru_l_rx_checks) / sizeof(subaru_l_rx_checks[0]);
 
+const uint16_t SUBARU_PARAM_FLIP_DRIVER_TORQUE = 1;
+bool subaru_flip_driver_torque = false;
```

Comment:
Let's specify that it's legacy. Maybe we should give legacy it's own file?

Reply:
Renamed the variable and moved legacy to separate file. 2020 global hybrid will also have separate safety mode so it made sense to separate legacy now